src/test/unit/profile-conversion.test.js

```diff
@@ -69,6 +69,21 @@ describe('converting Linux perf profile', function() {
   });
 });
 
+describe('converting dhat profiles', function() {
+  it('should import a dhat profile', async function() {
+    const fs = require('fs');
+    const zlib = require('zlib');
+    const buffer = fs.readFileSync('src/test/fixtures/upgrades/dhat.json.gz');
+    const decompressedArrayBuffer = zlib.gunzipSync(buffer);
+    const text = decompressedArrayBuffer.toString('utf8');
+    const profile = await unserializeProfileOfArbitraryFormat(text);
+    if (profile === undefined) {
+      throw new Error('Unable to parse the profile.');
+    }
+    expect(profile).toMatchSnapshot();
```

Comment:
This snapshot file is ridiculous at +31,013 line. The chrome one is equally ridiculous. I wonder if we shouldn't add a binary attribute to that specific snapshot test, that way it will be ignored by git line diffing.